platform/src/default.rs

```diff
@@ -134,4 +143,10 @@ impl Platform for DefaultPlatform {
         print!("{str}");
         Ok(())
     }
+
+    fn get_cert_validity(&mut self) -> Result<(&str, &str), PlatformError> {
+        let not_before = "20230227000000Z";
+        let not_after = "99991231235959Z";
```

Comment:
nit: Would be more clear to make these constants at the top of the Platform impl

Reply:
Done